src/components/donation/Steps/Submit/DAFCheckout/toPlatformValues.ts

```diff
@@ -0,0 +1,21 @@
+export function toPlatformValues(
```

Comment:
@SovereignAndrey, there's a case where on our donate form user inputs say `100`, we then suggest (can't be enforced) this to the DAFpay form 
<img width="378" alt="image" src="https://github.com/user-attachments/assets/f2d84dac-b8c8-4712-be1b-093d58981580">

but user could change this to say `120` or  `90`. I created this (`toPlatformValues`) helper to scale up or scale down tip, feeAllowance on this cases - kindly see input cases in `toPlatformValues.test.ts`